### PR TITLE
[F] Handle conflicting files

### PR DESCRIPTION
**Pull request by hectorhuertas**

### CODE REVIEW

File: src/main/gen/index.ts [hunk 1]
```diff
@@ -56,11 +55,19 @@ async function _handler(args: any): Promise<void> {
...
-  const {paths, editor} = await templates(args);
+  const {logs, editor} = await templates(args);
 
   if (verbose) {
-    Object.keys(paths).forEach(k => {
-      console.log(`${chalk.green('Creating:')} ${k}`);
+    logs.forEach(log => {
+      if (log.type === 'info') {
+        console.log(`${chalk.blue('Info:')} ${log.msg}`);
+      }
```
Comment (Istar-Eldritch) [inline src/main/gen/index.ts:64]: What is the info? What would be the purpose of it?

File: src/main/gen/templates.ts [hunk 2]
```diff
@@ -52,9 +81,32 @@ export default async function templates({mode, name, description}: TplOptions) {
...
+      const existingHash = crypto.createHash('md5').update(existing).digest('hex');
+      const templated = editor.read(`tmp/${tpl}`);
+      const templatedHash = crypto.createHash('md5').update(templated).digest('hex');
+
+      if (existingHash === templatedHash) {
+        ovewrite = false;
+        logs.push({type: 'skip', msg: tpl});
+      } else if (!noprompt) {
+        ovewrite = await confirmPrompt('confirm_ovewrite', `
+          File: "${tpl}" already exists, do you want to ovewrite it?`);
```
Comment (Istar-Eldritch) [inline src/main/gen/templates.ts]: There is a typo here. It should be `overwrite`.

**Review by Istar-Eldritch — CHANGES_REQUESTED** [review]

File: src/main/gen/index.ts (hunk 1 above)
Comment (hectorhuertas) [reply src/main/gen/index.ts:64]: Let the user know what task is going to run, like "generating templates"

File: src/main/gen/templates.ts (hunk 2 above)
Comment (hectorhuertas) [reply src/main/gen/templates.ts]: Thats how the cool kids write it today, xD

**Comment by hectorhuertas** [comment]
@Istar-Eldritch i think I addressed your changes. Can you review it?

**Review by Istar-Eldritch — APPROVED** [review]
LGTM

**Pull request merged**